src/mds/Server.cc

```diff
@@ -6718,6 +6719,10 @@ void Server::_committed_slave(MDRequestRef& mdr)
 
   ceph_assert(g_conf()->mds_kill_link_at != 8);
 
+  auto su = mdcache->get_uncommitted_slave(mdr->reqid, mdr->slave_to_mds);
+  if (su) {
```

Comment:
This isn't the right way. In a normal scenario, su will always be null.

Reply:
we should always call finish_uncommitted_slave() here